internal/pkg/levee/levee.go

```diff
@@ -50,7 +50,7 @@ func run(pass *analysis.Pass) (interface{}, error) {
 
 	for _, sources := range sourcesMap {
 		for _, s := range sources {
-			propagations[s.Node] = propagation.Dfs(s.Node, conf, taggedFields)
+			propagations[s.Node] = propagation.PropagateTaint(s.Node, conf, taggedFields)
```

Comment:
I think this name is better than `Dfs`, but it could still be improved. `propagation.PropagateTaint` as a whole is a bit repetitive. From the [Go blog](https://blog.golang.org/package-names):

> Avoid stutter. Since client code uses the package name as a prefix when referring to the package contents, the names for those contents need not repeat the package name. The HTTP server provided by the http package is called Server, not HTTPServer. Client code refers to this type as http.Server, so there is no ambiguity.

Reply:
(See also the internal `propagateTaint` function.)

Reply:
Any suggestion for the name?

Reply:
Just `Taint` would work IMO. I'd like to hear other reviewers' opinions on this though.

Reply:
I am not to be trusted on such things as naming.

That said, `Taint` feels like what affects the neighbors.  This is called on the source.  To me, that seems like `propagation.From(...)`, if we're going for maximally succinct.

While the exported `Dfs` and the recursive-step `dfs` names didn't seem like a conflict to me, I'm not sure I like an exported `Taint` and an unexported `taint`.  But that can also be solved by Michael's suggested for `taint -> taintNeighbors`.  So `Taint` here would probably be fine.

Reply:
Updated to 
Dfs -> Taint
dfs -> taint
visit -> taintNeighbors